package.json

```diff
@@ -17,32 +20,34 @@
   },
   "homepage": "https://github.com/PolinaBobrovnik/FrontCamp#readme",
   "devDependencies": {
-    "babel-core": "6.26.3",
-    "babel-loader": "8.0.4",
-    "babel-plugin-transform-runtime": "6.23.0",
-    "babel-polyfill": "6.26.0",
-    "babel-preset-es2015": "6.6.0",
-    "babel-preset-latest": "6.24.1",
-    "babel-preset-stage-0": "6.24.1",
-    "babel-register": "6.26.0",
-    "babelify": "7.3.0",
-    "del": "^2.2.2",
+    "@babel/core": "^7.1.6",
+    "@babel/plugin-transform-runtime": "^7.1.0",
+    "@babel/preset-env": "^7.1.6",
+    "babel-loader": "^8.0.4",
+    "babel-plugin-syntax-dynamic-import": "^6.18.0",
+    "babel-plugin-transform-runtime": "^6.23.0",
+    "clean-webpack-plugin": "^1.0.0",
+    "css-loader": "^1.0.1",
     "eslint": "^5.9.0",
     "eslint-config-airbnb-base": "^13.1.0",
     "eslint-plugin-import": "^2.14.0",
-    "gulp": "^3.9.1",
-    "gulp-babel": "^6.1.2",
-    "gulp-concat": "^2.6.1"
+    "eslint-loader": "^2.1.1",
+    "file-loader": "^2.0.0",
+    "html-loader": "^0.5.5",
+    "html-webpack-plugin": "^3.2.0",
+    "mini-css-extract-plugin": "^0.4.5",
+    "uglifyjs-webpack-plugin": "^2.0.1",
+    "url-loader": "^1.1.2",
+    "webpack": "^4.26.0",
+    "webpack-cli": "^3.1.2",
+    "webpack-dev-server": "^3.1.10"
   },
   "dependencies": {
+    "@babel/polyfill": "^7.0.0",
+    "@babel/runtime": "^7.1.5",
     "bootstrap": "4.0.0-alpha.6",
-    "browserify": "13.0.1",
     "es6-promise": "^4.2.5",
-    "gulp": "3.9.1",
-    "gulp-eslint": "^5.0.0",
-    "gulp-minify-css": "1.2.4",
-    "isomorphic-fetch": "^2.2.1",
-    "moment": "^2.13.0",
-    "vinyl-source-stream": "1.1.0"
+    "moment": "^2.22.2",
```

Comment:
please remove `^`

I suggest you to read meaning of symbols that can be used with package version